streaming/python/ray/streaming/config.py

```diff
@@ -0,0 +1,31 @@
+from __future__ import absolute_import
```

Comment:
What is the long term plan for Python streaming? Could you share any plans here? We would prefer to adopt an existing streaming API such as Flink or Beam (with extensions for Ray integration as an addition).

Are you planning to build on the existing Python streaming code or replace it?

Reply:
> What is the long term plan for Python streaming? Could you share any plans here? We would prefer to adopt an existing streaming API such as Flink or Beam (with extensions for Ray integration as an addition).
> 
> Are you planning to build on the existing Python streaming code or replace it?

Our final plan is to design a cross-language streaming api and flink has no such feature, but we may take Flink and Bean as references.

Reply:
We'll replace existing python streaming api with a new one. Currently we plan to adopt Flink API, with extensions such as schema and cross-lang streaming graph.
The long term plan is:
- improve  python api, which should be consistent with java.
- map python streaming graph to java streaming graph, then submit graph to java JobMaster.
- python JobWorker. It has same functions as java JobWorker, can be called by java JobMaster and run user function.
- support cross-lang streaming graph and schema. Any worker in graph can be a java or python worker. Schema can be used as data format when data exchanges between java/python workers.
- improve python/java api continually, such as add event time, state and so on.

#6184  has more high-level plans.